spec/models/card_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe Card, type: :model do
+  it 'check translation' do
+    card = Card.new(original_text: "дом")
+    expect(card.check_translation("вигвам")).to be false
+  end
+
+  it 'should check correct translation with strange case' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.